### PR TITLE
Fixing test issues, bringing in MEA_CCS code

### PR DESCRIPTION
**Pull request by bpaul4**
**NOTE:** When this PR is approved, you *must* do the following so it can be merged:
* Add the "approved" tag to the pull-request
* Trigger tests by pushing a commit or manually re-running in CircleCI

See [this wiki page](https://github.com/IDAES/idaes-dev/wiki/Github-Pull-Request-Workflow-with-Approval-Tag) for details.

----
## Changes proposed in this PR:
- fixes tests issues so QGESS and power_plant_capcost tests can run
- brings in MEA_CCS code and tests
- runs black to format the costing modules
- the base started a commit behind the head, hence the file tree shows changes that are already in main from 'Sphinx Documentation Build Issue'
- will check coverage if below threshold, otherwise should be fine

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

----

## Reviewer Checklist
* Reviewer 1
- [ ] Tests: do they pass, do they cover functionality, are they understandable?
- [ ] Documentation: is the code commented, are there user-level docs where needed?
* Reviewer 2
- [ ] Tests: do they pass, do they cover functionality, are they understandable?
- [ ] Documentation: is the code commented, are there user-level docs where needed?

### CODE REVIEW

**Comment by bpaul4** [comment]
It would be nice to run the tests here, @MAZamarripa are you able to approve the workflow?

File: idaes/models_extra/power_generation/costing/power_plant_capcost.py [hunk 1]
```diff
@@ -79,6 +79,12 @@
...
     in account 5.1.b and remains unchanged.
+
+    - 6.1 can be used to cost the MEA solvent-based carbon capture system, 
+    using the reference: Kangkang Li, Ashleigh Cousins, Hai You, Paul Feron,
+    Weilang Luo, Jian Chen (2016). Systematic study of aqueous monoethanolamine
+    (MEA)-based CO2 capture process: Techno-economic assessment of the MEA
+    process and its improvements. Applied Energy, 165, 648-659.
  """
 
 custom_costing_exponents = {
```
Comment (MAZamarripa) [inline idaes/models_extra/power_generation/costing/power_plant_capcost.py]: This file is getting too large, can we move the custom_costing_exponents to a new file and import these two dictionaries into the get_PP_costing method?

Comment (bpaul4) [reply idaes/models_extra/power_generation/costing/power_plant_capcost.py]: I'll move them into a new file

File: idaes/models_extra/power_generation/costing/tests/test_power_plant_capcost.py [hunk 2]
```diff
@@ -0,0 +1,767 @@
...
+    ###########################################################################
+
+    # coal flow rate
+    # accounts 1.x and 2.x are coal handling, preparation and feed
+    # accounts 4.x are for boiler BOP and foundations
+    coal_accounts = ["1.1", "1.2", "1.3", "1.4", "2.1", "2.2", "4.11", "4.15", "4.16"]
+    m.fs.boiler = pyo.Block()
+    m.fs.boiler.coal_mass_flow = pyo.Var(initialize=7238.95)  # tpd
+    m.fs.boiler.coal_mass_flow.fix()
+    QGESSCostingData.get_PP_costing(
```
Comment (MAZamarripa) [inline idaes/models_extra/power_generation/costing/tests/test_power_plant_capcost.py]: isnt this call now be something like this: 
    m.fs.boiler.costing = UnitModelCostingBlock(
        default={
            "flowsheet_costing_block": m.fs.costing,
            "costing_method": QGESSCostingData.get_PP_costing,
            "costing_method_arguments": {
                "cost_accounts": coal_accounts,
                "scaled_param": m.fs.boiler.coal_mass_flow,
                "units": "tpd",
                "tech": 2,
                "ccs": "A",
            },
        }
    )

Comment (bpaul4) [reply idaes/models_extra/power_generation/costing/tests/test_power_plant_capcost.py]: I've updated all calls like this one to use the new format.

**Comment by bpaul4** [comment]
@MAZamarripa the three pytest failures are known bugs, essentially numpy updated and broke some PySMO tests - Mayo is working on a fix. The pylint failure happens since the report() method is empty, how do you want to address this?

**Review by MAZamarripa — APPROVED** [review]

**Pull request merged**